Source/WebKit/PlatformGTK.cmake

```diff
@@ -63,14 +63,15 @@ list(APPEND WebKit_MESSAGES_IN_FILES
 
     WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf
 
-    WebProcess/gtk/GtkSettingsManagerProxy
+    WebProcess/glib/SystemSettingsProxyGLib
 )
 
 if (USE_GBM)
   list(APPEND WebKit_SERIALIZATION_IN_FILES Shared/glib/DMABufObject.serialization.in)
 endif ()
 
 list(APPEND WebKit_SERIALIZATION_IN_FILES
+    Shared/glib/ArgumentCodersGLib.serialization.in
```

Comment:
This is not a generic file for glib coders, it should be called SystemSettingsState.serialization.in